CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## 🛑 Breaking changes 🛑
 - Move fanout consumers to fanoutconsumer package (#2615)
+- Refactored `consumererror` package, eliminating `PartialError` type (#TBD)
```

Comment:
@Aneurysm9 not sure, do you think `consumererror[s]` is better name?

Reply:
It doesn't look like any other packages are plural, so I think sticking with `consumererror` would be best.  If a broader effort to pluralize package names were desired it should be undertaken separately.

Reply:
To be honest the only reason for this to be separate is the "errors" package in golang

Reply:
But happy with what we have, probably better to stop bikeshedding :)